wallet/src/main/java/com/android/identity/issuance/remote/StorageImpl.kt

```diff
@@ -48,6 +51,15 @@ internal class StorageImpl(
                 null,
                 null
             )
+            // TODO: Older OS versions don't support setting the cursor window size.
+            //  What should we do with older OS versions?
```

Comment:
I don't know how to answer this but it seems quite important :D

Reply:
I'm working on ideas to address this. Happy to discuss offline.